### PR TITLE
Feature/recipe list

### PR DESCRIPTION
**Pull request by Luxa23**

### CODE REVIEW

File: components/RecipeForm.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+export default function RecipeForm() {
+  return (
+    <>
+      <StyledForm>
+        <label>
+          <StyledInput
+            type="text"
+            size="30"
+            placeholder="Add recipe title"
+          ></StyledInput>
```
Comment (TheMarv) [inline components/RecipeForm.js:14]:
```suggestion
          />
```

use a self-closing tag

File: pages/index.js [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+import RecipeForm from '../components/RecipeForm';
+
 export default function Home() {
-  return <h1>my Recipes</h1>;
+  return (
+    <>
+      <ListEntry>Strawberry Cake</ListEntry>
+      <ListEntry>Chocolate Muffins</ListEntry>
+      <ListEntry>Coffee Cake</ListEntry>
+      <RecipeForm></RecipeForm>
```
Comment (TheMarv) [inline pages/index.js:10]:
```suggestion
      <RecipeForm />
```

use self-closing tags

**Review by TheMarv — CHANGES_REQUESTED** [review]
You might want to rethink the ListEntry component. Unless you want to add more to might, perhaps you should change it to a ul with li instead. If you plan to add more to the ListEntry later on it's fine.

File: components/ListEntry.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { StyledH2, StyledListEntry } from '../components/StyledComponents';
+
+export default function ListEntry({ children }) {
+  return (
+    <StyledListEntry>
+      <StyledH2>{children}</StyledH2>
+    </StyledListEntry>
```
[the remark this answers is not in the document]
Comment (Luxa23) [reply components/ListEntry.js:7]: since it is the title, yes

**Pull request merged**